libmcount/internal.h

```diff
@@ -440,9 +447,11 @@ struct mcount_disasm_info {
 	struct sym		*sym;
 	unsigned long		addr;
 	unsigned char   	insns[64];
+	struct cond_branch_info cond_info[MAX_COND_BRANCH];
 	int			orig_size;
 	int			copy_size;
 	bool			modified;
+	uint8_t		nbr_cond_branch;
```

Comment:
Please fix the indentation.  And we use "nr" for numbers like in Linux kernel, or "num" looks fine.  Also how about calling it "nr_branch" according to the suggestion above.